task4.2.js

```diff
@@ -0,0 +1,17 @@
+let PrUser = {
+    print() {
+        console.log(`Name: ${this.firstName}; Last Name: ${this.lastName}; age: ${this.age}`);
+    }
+}
+
+
+function User(firstName, lastName, age) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.age = age;
+    this.__proto__ = PrUser;
```

Comment:
This is a bad style doing this. Rewrite please.

Reply:
does it concern the 12 line or whole constructor function?

Reply:
12 line